src/drivers/px4io/CMakeLists.txt

```diff
@@ -68,7 +68,7 @@ file(RELATIVE_PATH fw_io_bin_relative ${CMAKE_CURRENT_BINARY_DIR} ${fw_io_bin})
 set(fw_io_bin ${fw_io_bin} PARENT_SCOPE)
 
 add_custom_command(OUTPUT ${fw_io_bin}
-	COMMAND mkdir -p ${PX4_BINARY_DIR}/ROMFS/genromfs/extras/
+	COMMAND mkdir -p ${PX4_BINARY_DIR}/genromfs/${config_romfs_root}/extras/
```

Comment:
@dagar: Is there a reason why you changed the path to where the px4io.bin file is placed? It is now placing the px4io.bin file in `ROMFS/genromfs/px4fmu_common/extras/`. I think it should be placed directly in `ROMFS/genromfs/extras/`.

Reply:
I believe it's really only an arbitrary temp directory to point the genromfs tool at. ${PX4_BUILD_DIR}/ROMFS is the cmake build working directory, so I explicitly chose a path outside of that. Let's just make sure the px4io binary ultimately ends up in ROM.

Reply:
I understand that... but... what you have done is set it up so that the `extras/` directory is now buried one level deeper within a `px4fmu_common` directory in the final ROMFS that ends up on the FMU.

The old ROMFS directory tree looked like this
```
|- /dev/
|- /etc/
  |- init.d/
  |- mixers/
  |-extras/
    |- px4io.bin
|- /fs/
```
Your build is generating the following:
```
|- /dev/
|- /etc/
  |- init.d/
  |- mixers/
  |-px4fmu_common
    |-extras/
      |- px4io.bin
|- /fs/
```

I guess you want it to still result in the old directory structure ON the Px4?

Reply:
Yep, that is a blatant late night hacking error. The final generated ROMFS that's embedded should be identical.

Reply:
Ok, good. We are on the same page, then. :-)

I'll merge in your changes, fix the path bug and then we can have another look at the final PR.